doc/dev/perf_counters.rst

```diff
@@ -200,3 +200,50 @@ The actual dump is similar to the schema, except that average values are grouped
    }
  }
 
+Labeled Perf Counters
+---------------------
+
+A Ceph daemon has the ability to emit a set perf counter instances with varying labels. These counters are intended for visualizing specific metrics in 3rd party tools like the Prometheus and Grafana.
+
+For example, the below counters show put and get byte statistics for different users on different buckets::
+
+  {
+    "Bucket:bkt1:User:user1:": {
```

Comment:
I think the doc should be modified given the recent change in the labeled perf dump format.